app/lib/actions/katello/repository/sync.rb

```diff
@@ -49,7 +49,7 @@ def plan(repo, options = {})
 
               contents_changed = skip_metadata_check || output[:contents_changed]
 
-              plan_action(Katello::Repository::IndexContent, :id => repo.id, :contents_changed => contents_changed)
+              plan_action(Katello::Repository::IndexContent, :id => repo.id, :force_index => contents_changed)
```

Comment:
i think here, we really just want:  :force_index => skip_metadata_check
I don't think we want contents_changed to affect it or not (relying on the timestamp for that)